docs/protocol-methods.rst

```diff
@@ -1055,3 +1055,106 @@ Identify the client to the server and negotiate the protocol version.
 
   ["ElectrumX 1.2.1", "1.2"]
   "ElectrumX 1.2.1"
+
+masternode.announce.broadcast
+--------------
+
+Pass through the masternode announce message to be broadcast by the daemon.
+
+Whenever a masternode comes online or a client is syncing, they will send this message which describes the masternode entry and how to validate messages from it.
+
+**Signature**
+
+  .. function:: masternode.announce.broadcast(signmnb)
+
+  * *signmnb*
+
+    Signed masternode broadcast message.
+  
+**Result**
+
+  True if the message was broadcasted succesfully otherwise False.
+
+masternode.subscribe
+--------------
+
+Returns the status of masternode.
+
+**Signature**
+
+  .. function:: masternode.subscribe(collateral)
+
+  * *collateral*
+
+    A masternode collateral is a transaction with a specific amount of coins, it's also known as a masternode identifier.
```

Comment:
So what is passed?  A transaction ID, or a transaction?  In hex?  How does it know which output is relevant?

Please add an example in a follow-up.